src/navigation/gate.py

```diff
@@ -173,6 +173,7 @@ def evaluate(self, ud):
             self.context.rover.get_pose(in_odom_frame=True),
             self.STOP_THRESH,
             self.DRIVE_FWD_THRESH,
+            use_odom=self.context.use_odom,
```

Comment:
Why do you use "use_odom=" here if the parameter is called "in_odom" in get_drive_command? This is also in partial_gate.py, search.py, and approach_post.py.

Reply:
nice catch, this is a mistake I'll fix it